charts/fluent-bit/templates/configmap-dashboards.yaml

```diff
@@ -0,0 +1,16 @@
+
+{{- if .Values.dashboards.enabled -}}
+{{- range $path, $_ :=  .Files.Glob "dashboards/*.json" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dashboard-{{ trimSuffix ".json" (base $path) }}
+  labels:
+    {{- include "fluent-bit.labels" $ | nindent 4 }}
+    grafana_dashboard: "1"
```

Comment:
Is the label key configurable in the grafana sidecar?

if so we should make it configurable here.

Also the dashboard cm may not always live in the same namespace as the helm chart, WDYT about making namespace configurable for the dashboard?

Reply:
I will make the label customizable. But I would strongly advise against supporting writing this config map to a different namespace to the chart, the sidecar can be configured to search all namespaces.